README.md

```diff
@@ -48,6 +48,48 @@ To run tests via Docker:
 docker-compose run mozilla/fxa_customs_server npm test
 ```
 
+## Tagging Releases
+
+Unlike other FxA services,
+the customs-server includes some non-public code changes
+that are managed in a separate private repo:
+
+  https://github.com/mozilla/fxa-customs-server-private/
+
+When tagging a new release for deployment,
+it needs to be merged to the private repo
+and re-tagged with those changes in place.
+The process looks something like this:
```

Comment:
Fwiw, I think this would be more useful it was a shell script than a command listing in a readme.